metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -620,15 +620,26 @@ def _get_folder_browse_path_v2_entries(
         if include_current_folder:
             yield BrowsePathEntryClass(id=urn, urn=urn)
 
+    def _make_chart_urn(self, element_id: str) -> str:
+        urn_params: dict = {
+            "name": element_id,
+            "platform": self.source_config.platform_name,
+        }
+
+        if self.source_config.include_looker_element_in_platform_instance:
+            urn_params["platform_instance"] = self.source_config.platform_instance
+
+        return builder.make_chart_urn(**urn_params)
```

Comment:
just do `make_chart_urn(..., platform_instance=self.source_config.platform_instance if ... else None)`

Reply:
This method is called from three places so same logic need to be repeated at three different place. The make_chart_urn is checking whether `include_looker_element_in_platform_instance` is enabled then applying the platform instance.

Reply:
removed the **urn syntax